### PR TITLE
python: add workflow for starting HTTP server

### PR DESCRIPTION
**Pull request by chaychoong**

### CODE REVIEW

**Review by alokedesai — APPROVED** [review]
Looks great, thank you so much for contributing! I have a super small question about whether we should set a default value for the directory, but besides that LGTM 🚀

File: specs/python/start_server.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+command: "python3 -m http.server {{port}} -d {{directory}}"
+tags: ["python"]
+description: Start a simple HTTP server using Python 3, using the in-built http.server module.
+arguments:
+  - name: port
+    description: The port number that the server should listen on.
+    default_value: ~
+  - name: directory
+    description: The directory to which the server will serve files.
+    default_value: ~
```
[the remark this answers is not in the document]
Comment (chaychoong) [reply specs/python/start_server.yml]: agreed, made the change!

[the remark this answers is not in the document]
Comment (alokedesai) [reply specs/python/start_server.yml]: looks great, thank you!

**Comment by elviskahoro** [comment]
@chaychoong are you in the Discord? If so what's your username, so I can add the contributor role?

**Comment by chaychoong** [comment]
> @chaychoong are you in the Discord? If so what's your username, so I can add the contributor role?

chay#2987, thanks

**Comment by elviskahoro** [comment]
Added it!

**Pull request merged**